.eslintrc

```diff
@@ -8,7 +8,7 @@
 		"es6": true
 	},
 	"rules": {
-		"quotes": ["error", "double"],
+		"quotes": [2, "double"],
```

Comment:
Weird, according to [the documentation](http://eslint.org/docs/rules/quotes) this should work. In my eslint PR I converted everything from `2` -> `"error"` because that's what the documentation is using.